src/casp/services/api/server.py

```diff
@@ -25,8 +25,10 @@ def __log(s):
     print(f"{dt_string} - {s}")
 
 
-async def __get_embeddings(myfile) -> t.Tuple[t.List, "numpy.array"]:
-    r_text = await async_client.CASAPIAsyncClient.call_model_service(file_to_embed=myfile.read())
+async def __get_embeddings(myfile, model_system_name: str) -> t.Tuple[t.List, "numpy.array"]:
+    r_text = await async_client.CASAPIAsyncClient.call_model_service(
+        file_to_embed=myfile.read(), model_system_name=model_system_name
+    )
     # TODO: json isn't very efficient for this
```

Comment:
Is there a way to get the embeddings in a more reasonable format, e.g. serialized torch tensor or numpy array? tensor to json to dataframe to numpy sounds totally unncessary!

Reply:
Absulotely, created an [issue](https://github.com/cellarium-ai/cellarium-cloud/issues/95) for it